xcell/mappers/mapper_KiDS1000.py

```diff
@@ -0,0 +1,261 @@
+            e2_flag = 'e2'
+        else:
+            raise ValueError(f"Unknown mode {mode}")
+        return kind, e1_flag, e2_flag, mode
+
+    def _check_lite_exists(self, zbin):
+        if self.path_lite is None:
+            return False, None
+        else:
+            fname_lite = self.path_lite + f'KiDS1000_lite_cat_zbin{zbin}.fits'
+            return os.path.isfile(fname_lite), fname_lite
+
+    def _bin_z(self, cat, zbin):
+        z_key = 'Z_B'
+        z_edges = self.zbin_edges[zbin]
+        return ((cat[z_key] > z_edges[0]) &
+                (cat[z_key] <= z_edges[1]))
+
+    def _remove_additive_bias(self, cat):
+        sel_gals = cat['SG_FLAG'] == 1
+        if np.any(sel_gals):
+            e1mean = np.average(cat['e1'][sel_gals],
+                                weights=cat['weight'][sel_gals])
+            e2mean = np.average(cat['e2'][sel_gals],
+                                weights=cat['weight'][sel_gals])
+            cat['e1'][sel_gals] -= e1mean
+            cat['e2'][sel_gals] -= e2mean
+
+    def _remove_multiplicative_bias(self, cat, zbin):
+        sel_gals = cat['SG_FLAG'] == 1
+        cat['e1'][sel_gals] /= 1 + self.m[zbin]
+        cat['e2'][sel_gals] /= 1 + self.m[zbin]
+
+    def _get_gals_or_stars(self, kind='galaxies'):
+        cat_data = self.get_catalog()
+        sel = cat_data['SG_FLAG'] == self.sel[kind]
+        return cat_data[sel]
+
+    def get_signal_map(self, mode=None):
+        kind, e1f, e2f, mod = self._set_mode(mode)
+        if self.maps[mod] is not None:
+            self.signal_map = self.maps[mod]
+            return self.signal_map
+
+        # This will only be computed if self.maps['mod'] is None
+        file_name1 = \
+            f'KiDS1000_signal_map_{mod}_e1_zbin{self.zbin}_ns{self.nside}'
+        file_name2 = \
+            f'KiDS1000_signal_map_{mod}_e2_zbin{self.zbin}_ns{self.nside}'
+        # Add extension separately to respect text width < 78
+        file_name1 += '.fits.gz'
+        file_name2 += '.fits.gz'
+        read_lite1, fname_lite1 = self._check_lite_exists(file_name1)
+        read_lite2, fname_lite2 = self._check_lite_exists(file_name2)
+        if read_lite1 and read_lite2:
+            print('Loading bin{} signal map'.format(self.zbin))
+            e1 = hp.read_map(fname_lite1)
+            e2 = hp.read_map(fname_lite2)
+            self.maps[mod] = [-e1, e2]
+        else:
+            print('Computing bin{} signal map'.format(self.zbin))
+            data = self._get_gals_or_stars(kind)
+            wcol = data['weight']*data[e1f]
+            we1 = get_map_from_points(data, self.nside, w=wcol,
+                                      ra_name='RAJ2000',
+                                      dec_name='DECJ2000')
+            wcol = data['weight']*data[e2f]
+            we2 = get_map_from_points(data, self.nside, w=wcol,
+                                      ra_name='RAJ2000',
+                                      dec_name='DECJ2000')
```

Comment:
Just to confirm. The old variables `ALPHA_J2000` and `DELTA_J2000` exist in the catalog. E.g. for `ALPHA_J2000`: 

```
SExtractor named Centroid sky position right ascension (J2000) (deg) 
```
Are we sure `RAJ2000` and `DECJ2000` are the right ones?

Reply:
They should be the same to an absurd precision. But it's a good point, let me use the other ones so we don't have to change the test files.